tests/test_updated_esdl_post_process.py

```diff
@@ -0,0 +1,106 @@
+import os
+import sys
+from pathlib import Path
+from unittest import TestCase
+
+from mesido.esdl.esdl_parser import ESDLFileParser
+from mesido.workflows import EndScenarioSizingStagedHIGHS
+
+import numpy as np
+
+
+class TestUpdatedESDL(TestCase):
+
+    def test_updated_esdl(self):
+        """
+        Check that the updated ESDL resulting from the optmizer, is correct by using the PoCTutorial
+        and the Grow_workflow
+
+        Checks:
+        - That the correct number of KPIs have been added
+        - That the correct assets have been removed
+        - Check that all the assets have a state=ENABLED
```

Comment:
can we also check:
- The sizes of the pipes and the sizes of source
- can we check that the aggregation count of the asset is set appropiately (I think there might still be a bug there)
- Check that unit of the KPI

Reply:
Has been added

Reply:
looks good!